File: Java/AlgoritmoNumeroPrimo.java
public class AlgoritmoNumeroPrimo {
    public static boolean esPrimo(int num) {
        if (num <= 1) {
            return false;
        }
        if (num == 2) {
            return true;
        }
        for (int i = 2; i < num; i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        int num = 25; // ejemplo
        if (esPrimo(num)) {
            System.out.println(num + " es primo");
        } else {
            System.out.println(num + " no es primo");
        }
    }
}

File: Python/AlgoritmoNumeroPrimo.py
def es_primo(num):
    if num <= 1:
        return False
    if num == 2:
        return True
    for i in range(2, num):
        if num % i == 0:
            return False
    return True

num = 25  # ejemplo
if es_primo(num):
    print(f"{num} es primo")
else:
    print(f"{num} no es primo")